pkg/util/templates/templates.go

```diff
@@ -0,0 +1,68 @@
+package templates
```

Comment:
change to singular form
```suggestion
package template
```

Reply:
I don't have strong feelings about this, but since there is not one template in the package but it collects all commonly used templates, it feels right to give it a plural name.

Reply:
we can resolve this in that case, we might circle back to that sometime